io.github.tsmetana.mpk3-settings.yml

```diff
@@ -0,0 +1,40 @@
+app-id: io.github.tsmetana.mpk3-settings
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+finish-args:
+# show windows with Wayland
+  - --socket=wayland
+# show windows using X11, if Wayland is not available
+  - --socket=fallback-x11
+# share IPC namespace with the host (necessary for X11)
+  - --share=ipc
+# Midi
+  - --device=all
+command: mpk3-settings
+cleanup:
+  - /include
+  - '*.a'
+  - '*.la'
+modules:
+  - name: mpk3-settings
+    buildsystem: autotools
+    post-install:
+      - install -D -t /app/share/metainfo pkg/${FLATPAK_ID}.metainfo.xml
+      - install -D -t /app/share/icons/hicolor/scalable/apps pkg/${FLATPAK_ID}.svg
+      - install -D -t /app/share/applications pkg/${FLATPAK_ID}.desktop
```

Comment:
why aren't these installed my the `make install`?

Reply:
I don't know the autoconf this much. Do you have an example how to do it?

Reply:
Got it. Thanks for the tip.

Reply:
It will be fixed in 0.1.1